#messgae published by tcp_node when an M command is recieved

int8 commandNum
float32 posX
float32 posY
float32 posZ
float32 orientationW
float32 orientationX
float32 orientationY
float32 orientationZ